### PR TITLE
samples: suit: SUIT DFU tests trigger

### PR DESCRIPTION
**Pull request by jnsgsr**
Trigger SUIT DFU system integration tests on NCS PRs

### CODE REVIEW

File: .github/test-spec.yml [hunk 1]
```diff
@@ -517,3 +517,27 @@
...
+  - "subsys/dfu/*"
+  - "subsys/fw_info/**/*"
+  - "subsys/mgmt/**/*"
+  - "subsys/pcd/*"
+  - "subsys/net/lib/*fota*/**/*"
+  - "subsys/net/lib/download_client/**/*"
+  - "subsys/suit/**/*"
+
+"CI-suit-dfu-test-extended":
+  - "subsys/suit/**/*"
```
Comment (robertstypa) [inline .github/test-spec.yml:543]: Missing directories which should trigger extended scope:
  - "subsys/mgmt/suitfu/**/*"
  - "samples/suit/**/*"

File: .github/test-spec.yml [hunk 2]
```diff
@@ -517,3 +517,27 @@
       - "subsys/nrf_security/**/*"
       - "!subsys/nrf_security/doc/**/*"
       - "!subsys/nrf_security/*.rst"
+
+"CI-suit-dfu-test":
```
Comment (robertstypa) [inline .github/test-spec.yml:521]: Missing entries:
  - "subsys/mgmt/suitfu/**/*"
  - "subsys/nrf_rpc/**/*"
  - "samples/suit/**/*"
  - "tests/subsys/suit/**/*"
  - "lib/dk_buttons_and_leds/**/*"
  - "subsys/nrf_security/**/*"
I'm not sure if all added entries are valid and should trigger SUIT tests. For example trusted-firmware-m, is there something the modules/trusted-firmware-m that SUIT depends on? The sampe situation for subsys/bootloader and subsys/nonsecure. Can we have post-daily online review with the dev team to confirm what is required from their perspective?

**Review by robertstypa — CHANGES_REQUESTED** [review]

File: .github/test-spec.yml (hunk 2 above)
Comment (robertstypa) [reply .github/test-spec.yml:521]: Disscussion regaring trusted-firmware-m: https://github.com/nrfconnect/sdk-nrf-next/pull/2085#pullrequestreview-1786144402

Comment (jnsgsr) [reply .github/test-spec.yml:521]: Thanks, updated on post-daily meeting.

**Review by e-rk — APPROVED** [review]

**Review by adsz-nordic — APPROVED** [review]

**Review by kszromek-nordic — APPROVED** [review]

**Review by robertstypa — APPROVED** [review]

**Pull request merged**